packages/app-project/src/components/ProjectHeader/components/ProjectTitle/ProjectTitle.js

```diff
@@ -16,10 +17,25 @@ const StyledHeading = styled(Heading)`
   text-shadow: 0 2px 4px rgba(0,0,0,0.5);
 `
 
-function ProjectTitle ({ href, title }) {
-  return href
-    ? <StyledAnchor href={href}><Title text={title} /></StyledAnchor>
-    : <Title text={title} />
+function ProjectTitle (props) {
+  const { link, title } = props
+  let result = (
+    <StyledHeading color='white' margin='none' size='small'>
+      {title}
+    </StyledHeading>
+  )
+
+  if (link) {
+    result = (
+      <Link as={link.as} href={link.href} passHref>
+        <StyledAnchor>
+          {result}
+        </StyledAnchor>
+      </Link >
```

Comment:
Shouldn't the heading wrap the link, not the other way round? I'm not sure about the markup here.

Reply:
`h1` is flow content, which is a permitted tag inside an anchor: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#Properties

Reply:
In terms of interactivity, it doesn't make any difference here as it's laid out with flexbox. Under normal circumstances, it'd be the difference between making a `block` element clickable, or just the text.